acceptance/testdata/search/search-issues.txtar

```diff
@@ -0,0 +1,19 @@
+# Create a repository with a file so it has a default branch
+exec gh repo create $ORG/$SCRIPT_NAME-$RANDOM_STRING --add-readme --private
+
+# Defer repo cleanup
+defer gh repo delete --yes $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Clone the repo
+exec gh repo clone $ORG/$SCRIPT_NAME-$RANDOM_STRING
+
+# Create an issue in the repo
+cd $SCRIPT_NAME-$RANDOM_STRING
+exec gh issue create --title 'Feature Request' --body 'Feature Body $RANDOM_STRING'
```

Comment:
`$RANDOM_STRING` is not going to expand to anything inside single quotes. Same on line 18. 

Maybe just do `--body $RANDOM_STRING`? It seems like using `-R` should scope the issues returned to only this new issue?

Reply:
Done 👍